app/cypress/support/commands.js

```diff
@@ -8,22 +8,33 @@ import getClientConfig from '$shared/utils/getClientConfig'
 
 const cache = {}
 
+const getWallet = (mnemonic) => {
+    const hdwallet = hdkey.fromMasterSeed(mnemonicToSeedSync(mnemonic))
+    return hdwallet.derivePath("m/44'/60'/0'/0").getWallet()
+}
+
 const generatePrivateKey = (mnemonic) => {
     if (!cache[mnemonic]) {
-        const hdwallet = hdkey.fromMasterSeed(mnemonicToSeedSync(mnemonic))
-        const wallet = hdwallet.derivePath("m/44'/60'/0'/0").getWallet()
-
+        const wallet = getWallet(mnemonic)
         cache[mnemonic] = wallet.getPrivateKey().toString('hex')
     }
-
     return cache[mnemonic]
 }
 
+const getAddress = (mnemonic = 'tester one') => {
+    if (!cache[mnemonic]) {
+        generatePrivateKey(mnemonic)
+    }
+
+    const wallet = getWallet(mnemonic)
+    return wallet.getAddressString()
+}
+
 Cypress.Commands.add('login', (mnemonic = 'tester one') => (
     new StreamrClient({
         restUrl: 'http://localhost/api/v1',
         auth: {
-            privateKey: generatePrivateKey(`auto generated user ${mnemonic}`),
+            privateKey: generatePrivateKey(mnemonic),
```

Comment:
I guess for testing it's enough to only two words for entropy?

Reply:
I think so. Other reason is that we can reference to other users easier with bare mnemonics. Of course we could wrap mnemonic handling with a function but I think this is easier and simpler.

Reply:
Yea, it's good like this 👍